### ISSUE TITLE
JDK 6 jobs on Travis are failing

### ISSUE THREAD

**Issue by SethTisue** [scala/scala-continuations#43]
see https://github.com/scala/scala-continuations/pull/42

**Comment by ashawley** [comment]
The buffer overflow is a hostname issue, but it looks like you have those bits in the Travis config.  For that reason, I'm at a loss of what it could be.

**Comment by SethTisue** [comment]
no longer relevant after #47
